open-metadata-implementation/access-services/subject-area/subject-area-api/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/properties/objects/term/Term.java

```diff
@@ -40,7 +42,15 @@
 import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.PUBLIC_ONLY;
 
 /**
- * TODO
```

Comment:
Having a TODO in class description Javadoc entry seems strange - would be better to start with a description of the class

Reply:
It looks like I am deleting the TODO here and replacing with text.